testbed/tests/metric_test.go

```diff
@@ -34,7 +34,7 @@ func TestMetric10kDPS(t *testing.T) {
 			testbed.NewOCDataReceiver(testbed.GetAvailablePort(t)),
 			testbed.ResourceSpec{
 				ExpectedMaxCPU: 29,
-				ExpectedMaxRAM: 65,
+				ExpectedMaxRAM: 70,
```

Comment:
I assume that there was a test failure for these.

Reply:
Affirm, there was.